slack_sdk/proxy_env_variable_loader.py

```diff
@@ -13,8 +13,11 @@ def load_http_proxy_from_env(logger: logging.Logger = _default_logger) -> Option
         or os.environ.get("HTTP_PROXY")
         or os.environ.get("http_proxy")
     )
-    if proxy_url is not None:
-        logger.debug(
-            f"HTTP proxy URL has been loaded from an env variable: {proxy_url}"
-        )
+    if proxy_url is None or len(proxy_url.strip()) == 0:
+        # If the value is an empty string, the intention should be unsetting it
+        if len(proxy_url.strip()) == 0:
+            logger.debug("HTTP proxy env variable is set but empty")
```

Comment:
Perhaps, telling what happened would be more informative:

```suggestion
            logger.debug("The Slack SDK ignored the proxy env variable as an empty value is set.")
```